auth/src/main/res/layout/fui_auth_method_picker_layout.xml

```diff
@@ -26,12 +27,23 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintVertical_bias="0.9">
+        app:layout_constraintVertical_bias="0.7">
 
         <LinearLayout
             android:id="@+id/btn_holder"
-            style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder" />
-
+            style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder"/>
     </ScrollView>
-
+    <TextView
+        android:id="@+id/main_tos_and_pp"
+        style="@style/FirebaseUI.Text.BodyText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/fui_field_padding_vert"
+        android:textColor="?android:textColorTertiary"
+        android:textIsSelectable="false"
+        android:gravity="center"
+        app:layout_constraintTop_toBottomOf="@id/container"/>
```

Comment:
Have you tested this with all numbers of buttons?  IMO it would be more natural for this to be constrained `Bottom_toBottomOf="parent"` so it's always exactly on the bottom.

Also make sure it looks OK in landscape.